### ISSUE TITLE
Development workflow question

### ISSUE THREAD

**Issue by joshuaiz** [internalfx/quickshot#46]
Hello,

I'm just getting into Shopify theme development and came across Quickshot so sorry for the noob questions but I just wanted to get some pointers on workflow.

My client has an active store which I was able to pull down into QS - that's all good. I set up a development store which I want to use for staging/dev to see theme changes before pushing to live.

However, when you create a new development store, it comes with only the Debut theme activated by default. When I tried to push the theme I downloaded from our live store, it pushes the files _into_ the current active theme, not as a separate theme. 

If so, how should I set up the dev store and a target in qs to use the correct theme? Should I add the theme to the dev store _first_ from the store admin, then add the target and push into that? Or is there another way?

The live site is using the Canopy theme and the theme structure looks like this:

Canopy
   - assets
   - config
   - mystore-myshopify-com-canopy [<-- this is the active theme]
      - assets
      - config
      - layout
      - locales
      - sections
      - snippets
      - templates
   - layout
   - locales
   - sections
   - snippets
   - templates

I'm not sure why the active theme is in a subfolder of the Canopy theme. Could this cause problems with QS? Is this how it is supposed to be set up on the live store? It doesn't feel right - I would think it would be better to have the active theme outside of another theme but maybe this is the way Shopify structures it if you work on a duplicate of your theme.

In short, I want to upload my current live theme to work locally, then push the theme to a development store so I can see changes before going live. 

Thanks!

**Comment by hughker** [comment]
@joshuaiz download ZIP of current Theme in liveStore then upload via Admin to devStore — then make "Active" — then setup QS to use that new theme. Makes sense?

**Comment by joshuaiz** [comment]
@hughker that makes perfect sense. Will give that a go...thanks so much.
